cardano-node/src/Cardano/Tracing/OrphanInstances/Shelley.hs

```diff
@@ -987,6 +996,12 @@ instance ToJSON (Alonzo.CollectError StandardCrypto) where
           , "error" .= String "NoCostModel"
           , "language" .= toJSON lang
           ]
+      -- TODO What do we want to show here?
```

Comment:
I think that `"kind" .= String "PlutusTranslationError"` would make sense.

The `_` has type `TranslationError`, which is an enum with four constructors. The `error` could mention which one it is:
* byron input in the presence of a plutus script
* byron output in the presence of a plutus script
* unnexpected/logic error regarding translating inputs
* unnexpected/logic error regarding translating redeemers